js/utils/constants.js

```diff
@@ -798,6 +798,47 @@
        */
       INITIAL_PLAY_STARTING: 'initialPlayStarting',
 
+      /**
+       * Notifies the player that the user has requested to play the previous video.
+       * Depending on the plugin being used, this could either be the previous video in
+       * a playlist, or the previously played Discovery video recommendation.
+       *
+       * @event OO.EVENTS#REQUEST_PREVIOUS_VIDEO
+       * @public
+       */
+      REQUEST_PREVIOUS_VIDEO: 'requestPreviousVideo',
```

Comment:
Do we want any success or failure messages? I'm guessing success could be covered by embedCodeChanged, but what about failure?

Reply:
So far this hasn't been necessary. The possible error scenarios are the standard errors that the player already handles.

Reply:
Sounds good